ha/setup/ha_setup.py

```diff
@@ -415,15 +415,15 @@ def process(self):
         s3_instances = ConfigCmd.get_s3_instance(machine_id)
         ios_instances = ConfigCmd.get_ios_instance(machine_id)
 
+        self._update_env(node_name, node_type, const.HA_CLUSTER_SOFTWARE, s3_instances, ios_instances)
+        self._fetch_fids()
+        self._update_cluster_manager_config()
+
         # fetch all nodes stonith config
         all_nodes_stonith_config: dict = {}
```

Comment:
Can you put some log message also 
to help to track

Reply:
Agree